pages/ksphere/konvoy/partner-solutions/jenkins/index.md

```diff
@@ -156,8 +176,21 @@ To begin installing Jenkins on Konvoy, you must download the `$JENKINS_HOME/jobs
     ```
 
     This command may take awhile. When it is finished, you should have the `jenkins_home/jobs` folder locally.
+  
+1. Download the `$JENKINS_HOME/nodes` directory to your work station:
```

Comment:
Have you tested this?

I dont think this would work. AFAIK, this file is almost always empty in DCOS Jenkins installations because we dont have statically configured agent nodes. Mesos plugin creates dynamic nodes and they don't show up here.

What we need is to copy the config.xml file in "jenkins_home" of DCOS Jenkins, which has config info about MesosCloud. We need to translate that into KubernetesCloud config and update Konvoy Jenkins' config.xml. Alternatively, before even installing Konvoy Jenkins we need to translate MesosCloud info to KubernetesCloud info and set that through values.yaml which is used by helm install.

Reply:
Note that just copying/migrating jobs and cloud configuration might not be enough. Credentials and other settings need to be copied as well. We should atleast mention the caveats. This is a decent guide about what all is involved in a migration: https://support.cloudbees.com/hc/en-us/articles/216241937-Migration-Guide-CloudBees-Jenkins-Platform-and-CloudBees-Jenkins-Team-


I would recommend to tackle migration section of the doc in a separate PR to avoid prolonging this PR.

Reply:
Alright, I will follow up with the nodes stuff in a later PR. Thanks.